docs/docusaurus/docs/core/connect_to_data/filesystem_data/_create_a_batch_definition/_examples/_file_partitioned_daily.py

```diff
@@ -0,0 +1,31 @@
+# <snippet name="docs/docusaurus/docs/core/connect_to_data/filesystem_data/_create_a_batch_definition/_examples/_file_partitioned_daily.py - full_example">
+from great_expectations import gx
+
```

Comment:
Most of the setup -- retrieving the Data Source and Asset -- are the same.  The major difference in the code is whether or not the Batch Definition takes in a `column` parameter or a `regex` parameter, and the corresponding values.  The explanation surrounding the directory vs the file Batch Definition is also different since each returns records on a different basis, so it seemed reasonable to maintain a separate script for each.  Since these are going to be put under test eventually, I wanted each to stand on their own (and to have the full process so that they could be put in the 'full example' tab as a full process without having to have two code blocks pulling from two files). Finally, having each under test would ensure that if a breaking change was made in either the File or Directory workflow without a similar change being made in the other, it would be caught when that particular version of the script ran in CI.

But yes: A lot of the time when we're dealing with examples toward the end of a process the bulk of the script is identical to previous ones with only a method or two and the corresponding parameters at the end having been changed.